app/locales/en/translations.js

```diff
@@ -279,7 +279,24 @@ export default {
     physician: 'Physician',
     assisting: 'Assisting',
     anesthesia: 'Anesthesia',
-    procedures: 'Procedures'
+    procedures: 'Procedures',
+    number: 'Number',
+    bill_date: 'Bill Date',
```

Comment:
i think quite a few of these are actually billing section only, shared between prices and invoices. can we nest those under billing.labels?